rj_gameplay/README.md

```diff
@@ -11,32 +11,44 @@ Stable code for gameplay.
 
 **Implementation - `rj_gameplay/`**
 
-Frequently changing code of gameplay .
+Frequently changing code of gameplay.
 
 **Tests - `tests/stp`**
 
 Unit tests for stp/.
 
 ## STP Framework
 
-Our gameplay library adapts the [STP
+Our gameplay library adopts the name of the [STP
 framework](https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.61.1972&rep=rep1&type=pdf)
-proposed by CMDragons.
+proposed by CMDragons, but is quite different.
 
-**Skill - `rj_gameplay/skill/`**
+**Play - `rj_gameplay/play/`**
 
-Atomic robot behavior which consists of a behavior tree and calls a ROS action.
-(Examples: Move, Kick)
+Handles full team behavior. Gets potential roles from a list of >=6 tactics,
+then assigns 6 of them at any time.
+
+(Examples: Basic Defense)
```

Comment:
Probably link to the file.

Reply:
Adding links would work well when viewing on GH, but not so much when from the command line. I can add filepaths instead so it's easy to find regardless.